ambari/src/main/java/io/brooklyn/ambari/server/AmbariServer.java

```diff
@@ -44,6 +46,9 @@
     PortAttributeSensorAndConfigKey HTTP_PORT =
             new PortAttributeSensorAndConfigKey("ambari.server.httpPort", "HTTP Port", "8080");
 
+    ConfigKey<Boolean> INSTALL_AGENT =
+            ConfigKeys.newBooleanConfigKey("ambari.server.install.agent","Install an ambari agent on same server", false);
```

Comment:
The method `agentOnServer()` uses the list of service components from the cluster so this configkey is not necessary and can be removed.